tools/docker-compose.yml

```diff
@@ -0,0 +1,44 @@
+
+  # run ngrok first to get access the PUBLIC HTTPS url --> Caddy. Caddy then routes the traffix to each port exposed on the 
+  # deployment driver container: 80 (the server 8081), 8080 (modm api server)
+  
+services:
+  caddy:
```

Comment:
Same here, we already have nginx in here so we might as well re-use that instead of using different component in development.